.travis.yml

```diff
@@ -1,13 +1,16 @@
 language: node_js
 node_js:
 - 8.11
+install:
+  - npm i -g npm@5.7.1 # this should be deleted once 5.7.X is out of "pre-release"
```

Comment:
Any reason we shouldn't just go straight to the latest release 6.0.1? I see that fixes to `npm ci` were mentioned in the [`5.8.0` release notes](https://github.com/npm/npm/releases/tag/v5.8.0). I know that the LTS version that ships with our node version is `5.6.0` but hopefully we can avoid more NPM weirdness if we stick to the later versions where the dust has settled on the `package-lock.json` fiasco.